src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -258,16 +258,8 @@ export class Accept extends Component<AcceptProps, AcceptState> {
   }
 }
 
-const AcceptRouteWrapper = props => (
-  <ContextConsumer>
-    {({ mediator }) => {
-      return <Accept {...props} mediator={mediator} />
-    }}
-  </ContextConsumer>
-)
-
 export const AcceptFragmentContainer = createFragmentContainer(
-  AcceptRouteWrapper,
+  trackPageViewWrapper(Accept),
```

Comment:
Rather than wrapping here (and all below), can we just move this to the top as a decorator to keep things consistent since we're already using decorators? 

```js
@track()
@trackPageViewWrapper()
export class Accept extends React.Component { ... }
```